plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/product-details/style.scss

```diff
@@ -68,6 +59,7 @@ html body .wp-block-woocommerce-product-details.is-style-minimal {
 				display: block;
 				border-left-width: 2px;
 				border-bottom-width: 0;
+				margin-bottom: 2px;
```

Comment:
Why do we need to add bottom margin here? :thinking:

Reply:
@Aljullu the bottom margin ensures consistency with the desktop style.

**Desktop**
![image](https://github.com/user-attachments/assets/bb580f15-9c24-449c-a831-85bbfe0bdbcb)

**Mobile**
![image](https://github.com/user-attachments/assets/896fc820-20bd-479e-bbc1-147b2223c898)

Reply:
Ah, it makes sense. Thanks for clarifying! :slightly_smiling_face: